### PR TITLE
use simple default email values

### PR DESCRIPTION
**Pull request by wolfaba**
... and let local MTA process process them

### CODE REVIEW

**Comment by wolfaba** [comment]
More info in https://bugs.bareos.org/view.php?id=1118

File: core/cmake/BareosSetVariableDefaults.cmake [hunk 1]
```diff
@@ -574,15 +574,15 @@ ENDIF()
...
 
 IF(NOT DEFINED job_email)
-   SET(job_email "root@localhost")
+   SET(job_email "root")
 ENDIF()
 
 IF(NOT DEFINED dump_email)
-   SET(dump_email "root@localhost")
+   SET(dump_email "root")
 ENDIF()
```
Comment (arogge) [inline core/cmake/BareosSetVariableDefaults.cmake]: We will need to discuss whether we want this to change.

File: core/cmake/BareosSetVariableDefaults.cmake [hunk 2]
```diff
@@ -574,15 +574,15 @@ ENDIF()
...
 
 IF(NOT DEFINED smtp_host)
-   SET(smtp_host "root@localhost")
+   SET(smtp_host "localhost")
```
Comment (arogge) [inline core/cmake/BareosSetVariableDefaults.cmake]: This looks good. This was probably an oversight when converting to CMake.

**Comment by arogge** [comment]
Could you please amend your commit so it complies to our [commit message guideline](https://docs.bareos.org/master/DeveloperGuide/gitworkflow.html#commit-message-guideline)  and references your bug?

**Comment by wolfaba** [comment]
> Could you please amend your commit so it complies to our commit message guideline and references your bug?

Hello arroge. I have tried to update the commit message, but I did it probably incorrectly and now, there are three commit messages. Should delete my fork, do the change again with correct commit message and create new pull request or could you accept these messages or change them to the correct one? Sorry, I am no developer, I have only little experience with github and git. Thank you.

File: core/cmake/BareosSetVariableDefaults.cmake (hunk 1 above)
Comment (wolfaba) [reply core/cmake/BareosSetVariableDefaults.cmake]: Ok, discuss it please. IMHO it should be either full qualified email address (which is impossible for generally generated package, only if someone compile bareos on his own host, he can enter full email address) or just "root", which should be then correctly completed by MTA. Thank you.

**Comment by arogge** [comment]
Sorry, I pushed the wrong branch and killed your PR that was not what I was trying to do.

**Comment by arogge** [comment]
And by doing this I also locked myself out of your branch...

**Comment by wolfaba** [comment]
ups, sorry for these trouble. Should I delete my fork, create new one, make correct single commit and new PR :-) ? Or do you make this change yourself direct in your repo bareos/bareos?

**Comment by arogge** [comment]
No need to apologize. *I* did it wrong.
If you could just apply the change you did to your bareos:master again, I could probably merge the change.

**Pull request closed without merging**